scripts/generate-icons.ts

```diff
@@ -0,0 +1,343 @@
+import fs from 'fs/promises';
+import { createCanvas, loadImage } from 'canvas';
+import { resolve } from 'path';
+import { getBrowser, releaseTarget } from './util';
+import colorLog from './log';
+import { PluginOption } from 'vite';
+
+/**
+ * Input folder containing the original svg icons in the source.
+ */
+const input = resolve('', 'src', 'icons');
```

Comment:
Are you sure the first empty `''` is needed - won't it resolve the same?

Reply:
You're right, fixed.